lib/ProductOpener/Routing.pm

```diff
@@ -574,20 +577,26 @@ sub analyze_request ($request_ref) {
 	}
 
 	# Return noindex empty HTML page for web crawlers that crawl specific facet pages
-	if (($request_ref->{is_crawl_bot} eq 1) and (defined $request_ref->{tagtype})) {
-		if ($request_ref->{tagtype} !~ /^brands|categories|labels|additives|nova-groups|ecoscore|nutrition-grades$/) {
-			# Only allow indexation of a selected number of facets
-			# Ingredients were left out because of the number of possible ingredients (1.2M)
-			$request_ref->{no_index} = 1;
-		}
-		elsif ($request_ref->{page} >= 2) {
-			# Don't index facet pages with page number > 1 (we want only 1 index page per facet value)
+	if ($request_ref->{is_crawl_bot} eq 1) {
+		if (defined $request_ref->{groupby_tagtype}) {
+			# $request_ref->{no_index} is set to 0 by default in init_request()
 			$request_ref->{no_index} = 1;
```

Comment:
Can we remove all the elsif and have only one if, so that we don't repeat $request_ref->{no_index} = 1; 4 times? We can have the comments inside the conditions